main/plugins/org.talend.sdk.component.studio-integration/jet_stub/generic/processor/processor_finally.javajet

```
@@ -44,5 +44,6 @@
             processor_<%=cid%>.stop();
         }
     } catch (final RuntimeException re) {
+        globalMap.put("<%=cid %>_ERROR_MESSAGE", re.getMessage());
         throw new TalendException(re, currentComponent, globalMap);
```

Should we throw this exception in other places instead of `throw e`?

---

It's (TalendException) a specific one and it handles in a specific way in generated code. 
I thought that the safest way here it'll be to catch a throw again the exception that we caught. It won't break anything in existed jobs.

I'm trying to understand why in TalendException#printStacktrace it's expected to put the exception message as ERROR_MESSAGES in globalMap.